content/en/docs/reference/helper-libraries/dra-driver-kubelet-plugin.md

```diff
@@ -0,0 +1,51 @@
+---
+title: DRA Resource Driver controller helper library
+description: Handles most of functionality common for all DRA resource drivers' kubelet-plugins.
+weight: 20
+---
+
+<!-- overview -->
+
+This library implements some of the functionality that the
+[Dynamic Resource Allocation](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/) (DRA)
+resource-drivers' kubelet-plugins need to implement, that is common for all of
+them, regardless of the actual resource managed or allocation logic. For example:
+
+- setting up non-blocking gRPC server on a Unix domain socket, and spawning a
+goroutine to handle requests for arbitrary services
+- registration with Kubelet
+- graceful shutdown
+
+<!-- body -->
+
+## Usage
+
+DRA kubelet-plugin helper code expects the resource driver to implement
+[NodeServer](https://pkg.go.dev/k8s.io/kubelet/pkg/apis/dra/v1alpha3#NodeServer) interface with
+the actual logic of how the resources should be prepared and unprepared:
```

Comment:
Same here: https://pkg.go.dev/k8s.io/dynamic-resource-allocation@v0.29.0/kubeletplugin

https://pkg.go.dev/k8s.io/kubelet/pkg/apis/dra/v1alpha3

Reply:
If we do that, we'd also want to ensure that the release docs team update the link each release.

For that and other reasons, I think it's more useful to put the details here. People who aren't contributing to Kubernetes would find this information useful.

Reply:
Replaced the interface quote with a link to _latest_ package version API reference. Only when the interface changes, then the documentation will need to be updated. A compromise.